learning/features/shading/shading_language.rst

```diff
@@ -377,41 +377,45 @@ CanvasItem (2D) - VertexShader
 +-----------------------------------+--------------------------------------------------------------------------------------------+
 | const float *TIME*                | Time (in seconds)                                                                          |
 +-----------------------------------+--------------------------------------------------------------------------------------------+
+| const bool *AT_LIGHT_PASS*        | Whether the shader is being run for a lighting pass (happens per affecting light)          |
++-----------------------------------+--------------------------------------------------------------------------------------------+
 
 CanvasItem (2D) - FragmentShader
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-+-------------------------------------+------------------------------------------------------------------------------+
```

Comment:
Yeah... I've tried to find a shorter wording by all means to avoid that cumbersome edit (and ugly diff) but nothing made sense.

Googling a bit I've found this: https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#tables

So in theory, MarkDown support ugly formatting for tables at the source code level. Maybe that would be a win.